prettier.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    printWidth: 80,
+    tabWidth: 4,
+    singleQuote: true,
+    arrowParens: 'avoid',
+    trailingComma: 'none'
+};
```

Comment:
Why not just use the standard Prettier style? `tabWidth: 4` is decidedly non-standard for JS, and the `trailingComma` is useful to make Git diffs less noisy. I don't think this boilerplate should be trying to enforce a non-standard code style.

Reply:
Fair. This is my personal preference but for the boilerplate we should probably stick with default prettier I suppose